modules/mono/glue/GodotSharp/GodotSharp/Core/Array.cs

```diff
@@ -25,16 +25,30 @@ protected override bool ReleaseHandle()
         }
     }
 
+    /// <summary>
+    /// Godot's <see cref="Array"/> is an array allocated inside of
+    /// the engine itself in C++. It mimics the features of arrays in
+    /// GDScript. Use it when interfacing with the engine. When arrays
+    /// only need to be interacted with from C#, prefer C# arrays.
```

Comment:
I would prefer something like this here:

```suggestion
    /// Wrapper around Godot's Array class, an array of Variant typed elements.
    /// Useful when interfacing with the engine. Otherwise .NET collections are recommended.
```

Reply:
Updated, not exactly the same as what you suggested, let me know what you think.